src/mod.ts

```diff
@@ -92,23 +63,11 @@ export function makeCylinderSurface(
 export function makeLine(
   length: number,
   noise1: Noise1Fn,
-  {
-    amplitude = defaultAmplitude,
-    frequency = defaultFrequency,
-    octaves = defaultOctaves,
-    persistence = defaultPersistence,
-    scale,
-  }: Partial<Options> = {},
+  options: Options,
 ): number[] {
   const field: number[] = new Array(length);
   for (let x = 0; x < length; x++) {
-    let value = 0.0;
-    for (let octave = 0; octave < octaves; octave++) {
-      const freq = frequency * Math.pow(2, octaves);
```

Comment:
Minor note. I think this was a typo and should be `octave`. It's changed to that in the `getLineNoiseValue` function